### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by stscicrawford**
Add 0.13.8 information to README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -115,6 +115,7 @@ Software vs DMS build version map
 
 | jwst tag | DMS build | CRDS_CONTEXT |   Date     |          Notes                           |
 | -------- | --------- | ------------ | ---------- | -----------------------------------------|
+|  0.13.8  | B7.3.1    | 0541         | 09/05/2019 | Patch release for Build 7.3.1            |
```
Comment (hbushouse) [inline README.md]: Technically isn't this a patch to B7.3, hence the Notes entry should read "Patch release for B7.3"?

Comment (stscicrawford) [reply README.md]: 'Patch for Build 7.3 released for Build 7.3.1'?

Comment (hbushouse) [reply README.md]: Minor quibble, but how about "Patch for Build 7.3 released **as** Build 7.3.1"?

Comment (stscicrawford) [reply README.md]: Done

File: README.md [hunk 2]
```diff
@@ -115,6 +115,7 @@ Software vs DMS build version map
 
 | jwst tag | DMS build | CRDS_CONTEXT |   Date     |          Notes                           |
 | -------- | --------- | ------------ | ---------- | -----------------------------------------|
+|  0.13.8  | B7.3.1    | 0541         | 09/05/2019 | Patch for Build 7.3 released as Build 7.3.1     |
```
Comment (hbushouse) [inline README.md]: Will the moderately brain-dead table formatter have a problem with the fact that the end of this line is not aligned with the rest of the table?

**Pull request merged**